### PR TITLE
BP-488 Delete Queries

### PR DESCRIPTION
**Pull request by julianweise**

### CODE REVIEW

File: src/SettingsView/EventQueryListItem.js [hunk 1]
```diff
@@ -20,13 +22,19 @@ class EventQueryListItem extends Component {
...
+					actionToPerform={this.deleteEventQuery}
+					ref={(input) => {this.confirmationMessage = input;}}>
+					Do you really want to delete this Query?
+				</ConfirmationMessage>
 				<IconButton onTouchTap={this.editEventQuery}>
 					<IconEdit/>
 				</IconButton>
-				<IconButton onTouchTap={this.deleteEventQuery}>
+				<IconButton
+					onClick={() => {this.confirmationMessage.handleOpen()}}>
```
Comment (MaximilianV) [inline src/SettingsView/EventQueryListItem.js]: Why not `onTouchTap` like line 33?

File: src/SettingsView/EventQueryListItem.js [hunk 2]
```diff
@@ -20,13 +22,19 @@ class EventQueryListItem extends Component {
...
 	}
 
-	static deleteEventQueryButton () {
+	deleteEventQueryButton () {
 		return (
 			<div>
+				<ConfirmationMessage 
+					actionToPerform={this.deleteEventQuery}
+					ref={(input) => {this.confirmationMessage = input;}}>
+					Do you really want to delete this Query?
```
Comment (MaximilianV) [inline src/SettingsView/EventQueryListItem.js]: Do we want to make this configurable?

File: src/SettingsView/EventType.js [hunk 3]
```diff
@@ -29,13 +29,18 @@ class EventType extends ConnectionComponent {
 
 	render() {
 		const allFetches = PromiseState.all([this.props.queries, this.props.attributes]);
+		const optionalActions =this.props.deleteQueryeResponse;
```
Comment (MaximilianV) [inline src/SettingsView/EventType.js]: Missing whitespace after `=`.
Typo in `deleteQueryResponse`.

**Review by MaximilianV — CHANGES_REQUESTED** [review]

File: src/SettingsView/EventType.js [hunk 4]
```diff
@@ -29,13 +29,19 @@ class EventType extends ConnectionComponent {
 
 	render() {
 		const allFetches = PromiseState.all([this.props.queries, this.props.attributes]);
+		const optionalActions = this.props.deleteQueryeResponse;
+		console.log(optionalActions);
```
Comment (hpibptlab) [inline src/SettingsView/EventType.js]: MAJOR Remove this logging statement. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS2228)

**Comment by hpibptlab** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Review by MaximilianV — APPROVED** [review]

**Pull request merged**